<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StageEditArea</class>
 <widget class="QWidget" name="StageEditArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton
    {    
     background-color: rgb(247,247,247); 
        border-style: outset; 
  border-radius: 10px; 
    padding:0px;     
        border-width: 2px; 
      border-color: #99b3d3; 
	font: 75 12pt &quot;Agency FB&quot;;
color:rgb(77,77,77);

     }

    QPushButton:hover
    {    
	font: 75 14pt &quot;Agency FB&quot;;
      border-color: gray; 
     border-width: 3px; 

     }

    QPushButton:pressed:checked
    {    
		font: 75 16pt &quot;Agency FB&quot;;
      border-color: black; 
     border-width: 3px; 

     }

    QPushButton:checked
    {    
		font: 75 16pt &quot;Agency FB&quot;;
      border-color: black; 
     border-width: 3px; 


     }



</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wBg" native="true">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#wBg{background-color: rgb(207, 226, 243);}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="4" column="0">
       <widget class="QPushButton" name="btnRemoveLayer">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);image: url(:/button/remove.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QWidget" name="wBottom" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="3" colspan="3">
       <widget class="QScrollArea" name="scrollArea">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>198</width>
           <height>590</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="1" column="0">
           <widget class="LayerSelector" name="wLayerSelector" native="true">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>54</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="btnBack">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/button/back-icon.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3" rowspan="4">
       <widget class="QWidget" name="wBody" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(242, 242, 242);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>11</number>
         </property>
         <property name="bottomMargin">
          <number>11</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QStackedWidget" name="wStackWork">
              <property name="styleSheet">
               <string notr="true">QWidget#wContent{background-color: rgb(239, 239, 239);}</string>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="pageWork">
               <property name="styleSheet">
                <string notr="true">QWidget#pageWork{background-color: rgb(242, 242, 242);}</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="2" column="1">
                 <widget class="QWidget" name="widget_5" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QPushButton" name="btnLayerBg">
                     <property name="font">
                      <font>
                       <family>Agency FB</family>
                       <pointsize>12</pointsize>
                       <weight>9</weight>
                       <italic>false</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>底圖</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="btnDelLayerBg">
                     <property name="font">
                      <font>
                       <family>Agency FB</family>
                       <pointsize>12</pointsize>
                       <weight>9</weight>
                       <italic>false</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>清除底圖</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="5" rowspan="3">
                 <widget class="QWidget" name="wEditArea" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_3">
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QTabWidget" name="tabWidget">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>22</pointsize>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgba(195, 75, 0,0);</string>
                     </property>
                     <property name="currentIndex">
                      <number>0</number>
                     </property>
                     <widget class="LayerEditor" name="wStyle">
                      <attribute name="title">
                       <string>屬性</string>
                      </attribute>
                     </widget>
                     <widget class="LayerAction" name="wAction">
                      <attribute name="title">
                       <string>行為</string>
                      </attribute>
                     </widget>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="7">
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="0">
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="2" rowspan="3">
                 <widget class="DisplayWidget" name="wDisplay" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>540</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" rowspan="2">
                 <widget class="LayerAddContent" name="wAdd" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>200</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnAddLayer">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/button/add.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LayerSelector</class>
   <extends>QWidget</extends>
   <header>LayerSelector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DisplayWidget</class>
   <extends>QWidget</extends>
   <header>DisplayWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayerEditor</class>
   <extends>QWidget</extends>
   <header>LayerEditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayerAddContent</class>
   <extends>QWidget</extends>
   <header>LayerAddContent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayerAction</class>
   <extends>QWidget</extends>
   <header>LayerAction.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
